### ISSUE TITLE
Put URL and body before headers

### ISSUE THREAD

**Issue by yDelouis** [mrmike/Ok2Curl#51]
Hi and thanks for the lib,

I'm using this lib in my project to have HTTP calls logged and be able to reproduce them and import them in Postman. It works great !
But it's quite hard to find the call we are looking for because every request log start with the same text  : 
```
curl -X GET -H "Authorization:Bearer <a very looonnngg token>" -H "<some more headers>: <some more headers values>" "https://<the url we are looking for>"
```
So we are still using a "standard" OkHttp logger to get the URL readable easily.

Would it be possible to change the order of arguments in the curl command to : 
- the method
- the url
- the body
- headers

Ex : 
`curl -X GET "https://example.com/endpoint" -d "a body" -H "header1: foo" -H "header2: bar"`

**Comment by disparate** [comment]
@yDelouis 
That what works for me in most cases:
```kt
val headerIndex = message.indexOf("-H")
val urlIndex = message.indexOf("\"http")
val start = message.substring(0, headerIndex)
val headers = message.substring(headerIndex, urlIndex)
val url = message.substring(urlIndex)
Log.d("Curl", "$start$url$headers")
```

**Comment by yDelouis** [comment]
@disparate Thanks for your help, I didn't think of parsing the log string.
I'd still like that it is done by the lib. But in the meantime, I will add this !

**Comment by flocsy** [comment]
+1

**Comment by flocsy** [comment]
see a possible solution: https://github.com/mrmike/Ok2Curl/pull/55

**Comment by C2H6O** [comment]
@disparate can you merge @flocsy's PR in? Is there a reason not to?

**Comment by mrmike** [comment]
@C2H6O I've merged PR and released version 0.7.0.
